<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>506</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Castellar</family>
    <pointsize>8</pointsize>
    <weight>50</weight>
    <italic>false</italic>
    <bold>false</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow_Menu</string>
  </property>
  <property name="styleSheet">
   <string notr="true">color: rgb(255, 185, 64);
background-color: rgb(4, 56, 25);
font: 8pt &quot;Castellar&quot;;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="BtnTablaPersonas">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>290</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lucida Calligraphy</family>
      <pointsize>11</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
      <underline>true</underline>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 30);
font: 11pt &quot;Lucida Calligraphy&quot;;</string>
    </property>
    <property name="text">
     <string>Tabla Personas</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BtnTablaProductos">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>290</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lucida Calligraphy</family>
      <pointsize>11</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
      <underline>true</underline>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 30);
font: 11pt &quot;Lucida Calligraphy&quot;;</string>
    </property>
    <property name="text">
     <string>Tabla Productos</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BtnGenerarArchivo">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>380</y>
      <width>271</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lucida Calligraphy</family>
      <pointsize>11</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
      <underline>true</underline>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 30);
font: 11pt &quot;Lucida Calligraphy&quot;;</string>
    </property>
    <property name="text">
     <string>Generar Archivo Reporte</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BtnSalir">
    <property name="geometry">
     <rect>
      <x>454</x>
      <y>422</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(131, 255, 247);</string>
    </property>
    <property name="text">
     <string>Salir</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>321</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Castellar</family>
      <pointsize>8</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
      <underline>false</underline>
      <stylestrategy>PreferDefault</stylestrategy>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:16pt; font-weight:600; color:#ffffff;&quot;&gt;Menu para ingreso &lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:16pt; font-weight:600; color:#ffffff;&quot;&gt;de informaci√≥n&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>550</width>
     <height>19</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
